spacy/tests/doc/test_span.py

```diff
@@ -434,3 +439,59 @@ def test_span_with_vectors(doc):
     # single-token span with vector
     assert_array_equal(ops.to_numpy(doc[10:11].vector), [-1, -1, -1])
     doc.vocab.vectors = prev_vectors
+
+def test_span_sents(doc, doc_not_parsed) :
```

Comment:
Could you rewrite this check using `@pytest.mark.parametrize`? Basically you'd define the offsets (we can hardcode `len(doc)`) and hardcode the expected number of sentences, and you'd only have to write the `assert` logic once.

The remaining part of this test can then be run as a separate function.

Reply:
Done